### PR TITLE
made correction to the code, no tags are hanging

### PR DESCRIPTION
**Pull request by BalrajDhakad**
Fixed the hanging tags and some extra "/".

## Fixes
resolves issue #15 
- #15

## Description
In the resource.html file there were some hanging tags and some extra "/" that have all been corrected.

## Screenshots
Corrected Code:
GitHub

## Checklist
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: docs/_data/languages.yml
Comment (Xaid-vfx) [inline docs/_data/languages.yml:1]: Removing languages with no resources is beyond the scope of issue #15.
Your PR should only have those changes that were documented and approved in the respective issue.

Comment (BalrajDhakad) [reply docs/_data/languages.yml:1]: Can you please tell me how can I undo my commits

Comment (BalrajDhakad) [reply docs/_data/languages.yml:1]: The main thing is I was not aware that if we do any commits on a branch with an existing PR those commits get added to that PR.

**Comment by BalrajDhakad** [comment]
I have made the required changes and undo the changes that I made in the "No resource available for some mentioned languages" commit. @Xaid-vfx

**Comment by possumbilities** [comment]
@BalrajDhakad There shouldn't be any changes showing in languages for this. Might want to look at it one more time.

**Comment by BalrajDhakad** [comment]
Actually, by mistake I made a few unwanted changes in which I removed few languages from the languages.yml file then I undoed those changes by readding those languages that I removed from the languages.yml file, this is done in the last commit that I have made in this PR with title "Removed Unwanted Changes"
@possumbilities

File: docs/_data/languages.yml [hunk 1]
```diff
@@ -51,6 +42,15 @@
 
 - name: Tagalog
   string: tl
+
```
Comment (possumbilities) [inline docs/_data/languages.yml:45]: There are still changes present here, the languages have been re-ordered around.

If there were no changes this file wouldn't show up here.

**Comment by BalrajDhakad** [comment]
It's because earlier due to some mistake understanding I removed the languages which has no resources available when we click on them, this was also an issue #80  that I was working on that time but was not able to change the branch and commited the changes that i made for that issue in this branch in which I am trying to merge in this PR.
Then I came to know about my mistake and I was not aware of any way to undo my commits that I have pushed so I made another change in order to add back the languages that I deleted.

@possumbilities 

I think the error is because the issue #80  that I was working on was solved by the PR # and now the changes that I pushed by mistake to this branch are the changes with the help of which the issue #80 was solved and later I removed those changes, so now there may be some conflicts between the main branch and this branch.
I will soon correct the error.

**Comment by BalrajDhakad** [comment]
@possumbilities 
I have made the languages.yml file same as that in the main branch of the codebase, so now there will be no problem if you merge this branch with the main branch and the issue #15 will also be solved.

**Comment by possumbilities** [comment]
I'm unsure this fixes all the hanging tags. From looking at #99 there's a number of areas where tags are hanging or done incorrectly that would satisfy #15, but this PR only targets changes in 1 file?

@BalrajDhakad Are you sure this is a complete fix?

**Comment by BalrajDhakad** [comment]
This is because there were no hanging or unclosed tags in any other file except resources.html and cc-licenses-poster.html.
I have gone through each and every line of code in each file and found error in these two only.
And as of my overview there are no more tags which are hanging.
@possumbilities

**Comment by BalrajDhakad** [comment]
Talking about #99, if you want me to make the code cleaner I will soon do it, but I am not able to understand that if the links are working properly then it is not needed to change any link in  the codebase. 
@possumbilities

**Comment by BalrajDhakad** [comment]
Okay, Not a problem.
@TimidRobot

**Pull request closed without merging**